openedx_caliper_tracking/tasks.py

```diff
@@ -7,21 +7,18 @@
 
 from celery.task import task
 from django.conf import settings
+from django.core.cache import cache
 from kafka import KafkaProducer
 from kafka.errors import KafkaError
-
 from openedx.features.ucsd_features.utils import send_notification
+from openedx_caliper_tracking.kafka_settings import KafkaSettings
 
 LOGGER = logging.getLogger(__name__)
-KAFKA_SETTINGS = settings.CALIPER_KAFKA_SETTINGS
 DEFAULT_FROM_EMAIL = settings.DEFAULT_FROM_EMAIL
-MAXIMUM_RETRIES = KAFKA_SETTINGS.get('MAXIMUM_RETRIES')
-ERROR_REPORT_EMAIL = KAFKA_SETTINGS.get('ERROR_REPORT_EMAIL')
-KAFKA_END_POINT = KAFKA_SETTINGS.get('END_POINT')
-KAFKA_TOPIC_NAME = KAFKA_SETTINGS.get('TOPIC_NAME')
+CACHE_KEY = 'IS_KAFKA_DELIVERY_FAILURE_EMAIL_SENT'
```

Comment:
@tasawernawaz 
`KAFKA` has been adjusted in the `key` as demanded :boom: :boom:  :sweat_smile: